.github/workflows/upgrade_test.yaml

```diff
@@ -0,0 +1,55 @@
+name: Upgrade functionality test
+
+on: 
+  push:
+    branches: [ upgrade-testing ]
+
+jobs:
+  
+  upgrade-functionality-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout action sources
+      uses: actions/checkout@v2.3.4
+
+    - name: Setup go
+      uses: actions/setup-go@v2.1.3
+      with:
+        go-version: '1.15.12'
+
+    - name: Setup Helm
+      uses: Azure/setup-helm@v1
+      with:
+        version: v3.3.4
+
+    - name: Setup Kind Clutser
+      uses: engineerd/setup-kind@v0.5.0
+      with:
+        config: kind.yaml
+        
+    - name: Setup kubectl & fetch node information
+      run: |
+        kubectl cluster-info --context kind-kind
+        kind get kubeconfig --internal >$HOME/.kube/config
+        kubectl get nodes
+        
+    - name: Dump system info
+      run: |
+        source ./test/upgrade_test/fission_objects.sh dump_system_info
+
+    - name: Install and configure previous stable fission
+      run: |
+        source ./test/upgrade_test/fission_objects.sh install_stable_release \
+        && create_fission_objects \
+        && test_fission_objects
```

Comment:
We don't need to test fission objects here as it's the stable one.

Reply:
We need to make sure that the functions created in stable version works before upgrading even if it stable.